File: namsor/csharp/src/OSEG.NamsorExamples/PhoneCodeGeoBatchExample.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Text.Json;

using App.Namsor.Api;
using App.Namsor.Client;
using App.Namsor.Model;

namespace OSEG.NamsorExamples;

public class PhoneCodeGeoBatchExample
{
    public static void Run()
    {
        var config = new Configuration();
        config.ApiKey = new Dictionary<string, string> {["api_key"] = "YOUR_API_KEY"};

        var personalNamesWithPhoneNumbers1 = new FirstLastNamePhoneNumberGeoIn(
            id: "e630dda5-13b3-42c5-8f1d-648aa8a21c42",
            firstName: "Teniola",
            lastName: "Apata",
            phoneNumber: "08186472651",
            countryIso2: "NG",
            countryIso2Alt: "CI"
        );

        var personalNamesWithPhoneNumbers = new List<FirstLastNamePhoneNumberGeoIn>
        {
            personalNamesWithPhoneNumbers1,
        };

        var batchFirstLastNamePhoneNumberGeoIn = new BatchFirstLastNamePhoneNumberGeoIn(
            personalNamesWithPhoneNumbers: personalNamesWithPhoneNumbers
        );

        try
        {
            var response = new SocialApi(config).PhoneCodeGeoBatch(
                batchFirstLastNamePhoneNumberGeoIn: batchFirstLastNamePhoneNumberGeoIn
            );

            Console.WriteLine(response);
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling SocialApi#PhoneCodeGeoBatch: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}

File: namsor/typescript-axios/src/PhoneCodeGeoBatchExample.ts
import axios, { AxiosError } from "axios";
import * as api from "namsor_client"

const configuration = new api.Configuration({
  apiKey: "YOUR_API_KEY",
});

const personalNamesWithPhoneNumbers1: api.FirstLastNamePhoneNumberGeoIn = {
  id: "e630dda5-13b3-42c5-8f1d-648aa8a21c42",
  firstName: "Teniola",
  lastName: "Apata",
  phoneNumber: "08186472651",
  countryIso2: "NG",
  countryIso2Alt: "CI",
};

const personalNamesWithPhoneNumbers = [
  personalNamesWithPhoneNumbers1,
];

const batchFirstLastNamePhoneNumberGeoIn: api.BatchFirstLastNamePhoneNumberGeoIn = {
  personalNamesWithPhoneNumbers: personalNamesWithPhoneNumbers,
};

new api.SocialApi(configuration).phoneCodeGeoBatch(
  batchFirstLastNamePhoneNumberGeoIn,
).then(response => {
  console.log(response.data);
}).catch((error: Error | AxiosError) => {
  console.log("Exception when calling SocialApi#phoneCodeGeoBatch:");
  axios.isAxiosError(error) ? console.log(error.message) : console.log(error);
});
